backend/src/app.ts

```diff
@@ -23,6 +23,7 @@ class App {
         this.routes(appInit.controllers);
         this.env();
         this.error();
+        require('./shared/passportHandler')
```

Comment:
Same here we should use import instead of require

Reply:
This is something we need to discuss, I have no idea how to implement this.